### DISCUSSION TITLE
Additional Language Support in Seamless Streaming

### DISCUSSION THREAD

**Discussion by serhanylmz** [facebookresearch/seamless_communication#307] [Ideas]
Hello, when reading the seamless paper, I can see that some languages are mentioned but are not available to use in the live demo on HF Spaces.

I was wondering if there's any way to make Seamless Streaming work with additional languages as mentioned in the paper, or if we'd need to train on our own data for that. I see great benefit in enabling this model to be used in such a way.

Best.

**Comment by annasun28** [comment]
@serhanylmz in the dropdown of `Output` in the live SeamlessStreaming demo, there should be 36 output speech languages. You should be able to switch between the 101 source speech languages without selecting anything. For the 96 text output languages (for S2T), you can add them to https://huggingface.co/spaces/facebook/seamless-streaming/blob/main/seamless_server/src/simuleval_agent_directory.py#L12-L19 if you just want to get text output (without the speech output).

**Reply by serhanylmz** [reply]
Thank you for the clarification!
